README.md

```diff
@@ -297,60 +281,209 @@ The complete list of transformations supported and their usage in ImageKit can b
+
+### Basic Examples
+```swift
+// https://ik.imagekit.io/your_imagekit_id/default-video.mp4?tr=h-400.00,w-400.00
+ImageKit.shared.url(
+    path: "default-video.mp4",
+    transformationPosition: TransformationPosition.QUERY
+)
+.height(height: 400)
+.width(width: 400)
+.create()
+```
 
-[Learn how to implement authenticationEndpoint](https://docs.imagekit.io/api-reference/upload-file-api/client-side-file-upload#how-to-implement-authenticationendpoint-endpoint) on your server.
+### Adaptive bitrate streaming
+To obtain the video URL with adaptive streaming, call `ImageKit.shared.url(...).setAdaptiveStreaming(...)` with a set of parameters defined below.
 
+| Parameter   | Type            | Description                                                                                                               |
+|:------------|:----------------|:--------------------------------------------------------------------------------------------------------------------------|
+| format      | StreamingFormat | Specifies the format for streaming video. Supported values for type are `StreamingFormat.HLS` and `StreamingFormat.DASH`. |
+| resolutions | [Int]      | Specifies the representations of the required video resolutions. E. g. 480, 720, 1080 etc.                                |
+
+Code example:
+```swift
+// https://ik.imagekit.io/your_imagekit_id/default-video.mp4/ik-master.m3u8?tr=sr-240_360_480_720_1080_1440_2160
+ImageKit.shared.url(
+    path: "default-video.mp4",
+    transformationPosition: TransformationPosition.QUERY
+)
+.setAdaptiveStreaming(
+    format: StreamingFormat.HLS,
+    resolutions: [240, 360, 480, 720, 1080, 1440, 2160]
+)
+.create()
+```
+
+
+## File Upload
+The SDK provides a simple interface using the `ImageKit.shared.uploader().upload(...)` method to upload files to the ImageKit Media Library. It accepts all the parameters supported by the [ImageKit Upload API](https://docs.imagekit.io/api-reference/upload-file-api/client-side-file-upload#request-structure-multipart-form-data).
 
 The `ImageKit.shared.uploader().upload(...)` accepts the following parameters
 
 | Parameter             | Type | Description                    |
 | :----------------| :----|:----------------------------- |
 | file | Data / UIImage / String | Required. 
 | fileName | String | Required. If not specified, the file system name is picked. 
-| useUniqueFileName  | Boolean | Optional. Accepts `true` of `false`. The default value is `true`. Specify whether to use a unique filename for this file or not. |
+| token | String | Required. the client-generated JSON Web Token (JWT), which the ImageKit.io server uses to authenticate and check that the upload request parameters have not been tampered with after the generation of the token. Refer this [guide](https://docs.imagekit.io/api-reference/upload-file-api/secure-client-side-file-upload#how-to-implement-authenticationendpoint-endpoint) to create the token below on the page. |
```

Comment:
Proper type casing after the word `Required`

```suggestion
| token | String | Required. The client-generated JSON Web Token (JWT), which the ImageKit.io server uses to authenticate and check that the upload request parameters have not been tampered with after the generation of the token. Refer this [guide](https://docs.imagekit.io/api-reference/upload-file-api/secure-client-side-file-upload#how-to-implement-authenticationendpoint-endpoint) to create the token below on the page. |
```

Reply:
Done